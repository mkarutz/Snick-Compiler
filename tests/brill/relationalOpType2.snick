proc main ()
    int x;
    bool y;

    x := 1;
    y := true;

    write x < y;
end
