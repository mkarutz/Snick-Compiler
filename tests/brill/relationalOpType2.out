Failure: Incompatible types in relational expression.
Compilation Failed
